### ISSUE TITLE
node_module files are cleaned

### ISSUE THREAD

**Issue by biemond** [athombv/node-athom-cli#158]
Hi, 

with https://www.npmjs.com/package/homey/v/2.0.9

I get this error , when rollback to 2.0.4 all is working again. 
```
2020-02-01 15:47:07 [log] [MyApp] philipsair is running...
2020-02-01 15:47:07 [log] [ManagerDrivers] [device] Device driver has been inited
2020-02-01 15:47:07 [log] [ManagerDrivers] [device2] Device driver has been inited
module.js:550
    throw err;
    ^

Error: Cannot find module 'index.js'
    at Function.Module._resolveFilename (module.js:548:15)
    at Function.Module._load (module.js:475:25)
    at Module.require (module.js:597:17)
    at Module.require (/opt/homey-client/system/manager/ManagerApps/bootstrap/sdk/v2/index.js:1:253)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/drivers/device/device.js:4:20)
    at Module._compile (module.js:653:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
```
can you look at the cleanup of node_modules or add an option to skip this.

thanks

**Comment by WeeJeWel** [comment]
Are the modules you’re using defined in your package.json file?

**Comment by biemond** [comment]
yes and no, this index.js is inside node_modules.  
like this https://github.com/biemond/com.athom.philipsair/blob/master/node_modules/index.js

it was supported

**Comment by WeeJeWel** [comment]
Your file index.js is sitting inside the node_modules folder, which is not the right place for it to be. You should move it up one level.

**Comment by biemond** [comment]
also did that, it was always ignored somehow, but let me try again with version 2.0.4, instead of 2.0.9
